src/index.html

```diff
@@ -15,8 +15,102 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <!-- <link rel="stylesheet" href="https://fonts.google.com/specimen/Roboto"> -->
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="header__logo-pic"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="header__item">
```